Automattic/jetpack
# Add Stats section that will be used in My Jetpack

## Proposed changes:

* Abstract the wrapper for product cards into its own component
* Moved "Status" to its own component - which is used to display the status (Active, Inactive, etc) of a specific product in a Card
* Add a new "Stats Section" component that looks like the highlight cards that are displayed on the Stats page. This component looks like the screenshot below

Card:
<img width="1020" alt="image" src="https://github.com/Automattic/jetpack/assets/5714212/5d6723a3-2325-4b88-8413-22618846b5f6">

Stats Section:
<img width="970" alt="image" src="https://github.com/Automattic/jetpack/assets/5714212/8f0fdc7a-5433-40bc-8c7c-4a5aa4b536b2">


### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Run this branch locally
* Run `jetpack build js-packages/storybook`
* Then `cd projects/js-packages/storybook && pnpm run storybook:dev`
* Visit the Storybook page and confirm that the Stats section matches the one proposed in the design
